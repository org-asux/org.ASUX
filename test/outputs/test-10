- - updatePet
  - myown_delete_api_key
  - string
  - '#/definitions/API'
- - updatePet
  - body
  - CustomObject
  - '#/definitions/Pet'
- - updatePet
  - petId
  - integer
  - '#/definitions/ID'
- - updatePet
  - petname
  - CustomObject
  - '#/definitions/User'
