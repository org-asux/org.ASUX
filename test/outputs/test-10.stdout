u-north-1 AZ no. 0 is: eu-north-1a AZ no. 1 is: eu-north-1b AZ no. 2 is: eu-north-1c  
ap-south-1 AZ no. 0 is: ap-south-1a AZ no. 1 is: ap-south-1b  
eu-west-3 AZ no. 0 is: eu-west-3a AZ no. 1 is: eu-west-3b AZ no. 2 is: eu-west-3c  
eu-west-2 AZ no. 0 is: eu-west-2a AZ no. 1 is: eu-west-2b AZ no. 2 is: eu-west-2c  
eu-west-1 AZ no. 0 is: eu-west-1a AZ no. 1 is: eu-west-1b AZ no. 2 is: eu-west-1c  
ap-northeast-2 AZ no. 0 is: ap-northeast-2a AZ no. 1 is: ap-northeast-2c  
ap-northeast-1 AZ no. 0 is: ap-northeast-1a AZ no. 1 is: ap-northeast-1c AZ no. 2 is: ap-northeast-1d  
sa-east-1 AZ no. 0 is: sa-east-1a AZ no. 1 is: sa-east-1c  
ca-central-1 AZ no. 0 is: ca-central-1a AZ no. 1 is: ca-central-1b  
ap-southeast-1 AZ no. 0 is: ap-southeast-1a AZ no. 1 is: ap-southeast-1b AZ no. 2 is: ap-southeast-1c  
ap-southeast-2 AZ no. 0 is: ap-southeast-2a AZ no. 1 is: ap-southeast-2b AZ no. 2 is: ap-southeast-2c  
eu-central-1 AZ no. 0 is: eu-central-1a AZ no. 1 is: eu-central-1b AZ no. 2 is: eu-central-1c  
us-east-1 AZ no. 0 is: us-east-1a AZ no. 1 is: us-east-1b AZ no. 2 is: us-east-1c AZ no. 3 is: us-east-1d AZ no. 4 is: us-east-1e AZ no. 5 is: us-east-1f  
us-east-2 AZ no. 0 is: us-east-2a AZ no. 1 is: us-east-2b AZ no. 2 is: us-east-2c  
us-west-1 AZ no. 0 is: us-west-1a AZ no. 1 is: us-west-1b  
us-west-2 AZ no. 0 is: us-west-2a AZ no. 1 is: us-west-2b AZ no. 2 is: us-west-2c AZ no. 3 is: us-west-2d  
